webrender/src/frame_builder.rs

```diff
@@ -2449,15 +2440,10 @@ fn adjust_corner_for_box_shadow(
 fn adjust_radius_for_box_shadow(
     border_radius: f32,
     spread_amount: f32,
-    spread_radius: f32,
 ) -> f32 {
-    // Adjust the shadow box radius as per:
-    // https://drafts.csswg.org/css-backgrounds-3/#shadow-shape
```

Comment:
why isn't this code following the CSS spec any longer?

Reply:
@mstange found that neither Chrome nor Gecko follow that part of the spec. He's opened a bug to remove or reconsider that text in the spec - https://github.com/w3c/csswg-drafts/issues/1900.